src/pages/Search.tsx

```diff
@@ -44,6 +44,17 @@ export default function Search() {
       <div>
         {data &&
           data.map((r) => <SearchResult result={r as Agent} key={r.id} />)}
+        <Card body border="light" className="shadow text-center">
```

Comment:
To match the flat style in the mobile app (note removing the `body` prop:
```suggestion
        <Card border="0" className="text-center">
```

Reply:
This style makes so much more sense to me. It smoothly meshes with the agent cards. And dropping the shadow adds enough distinction to differentiate it subtlety from the list of agent cards.